package/mediacenter-addon-osmc/files/DEBIAN/postinst

```diff
@@ -41,13 +41,16 @@ then
 	if ! is_rbp; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.pi* >/dev/null 2>&1; fi
 	if is_atv; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.updates >/dev/null 2>&1; fi
 	if ischroot; then OWNER="1000:1000"; else OWNER="osmc:osmc"; fi
-	chown -R $OWNER /usr/share/kodi/addons/script.module.elementtree
+	rm -rf /usr/share/kodi/addons/script.module.elementtree >/dev/null 2>&1
 	chown -R $OWNER /usr/share/kodi/addons/script.module.xmltodict
 	chown -R $OWNER /usr/share/kodi/addons/script.module.osmcsetting.*
 	chown -R $OWNER /usr/share/kodi/addons/service.osmc.settings
+	chmod +x /usr/bin/grab-logs
```

Comment:
Just chmod +x this in Git.

Reply:
If that works in the master git, fine, but I could not get those exec flags to stick here so I suspect @anxdpanic is finding the same.